package.json

```diff
@@ -20,15 +20,15 @@
   },
   "homepage": "https://github.com/KyberNetwork/smart-contracts#readme",
   "dependencies": {
-    "@nomiclabs/buidler-truffle5": "1.0.2",
+    "@nomiclabs/buidler-truffle5": "^1.1.2",
     "chai-as-promised": "7.1.1",
     "compare-versions": "3.5.1",
     "ganache-cli": "6.8.2",
     "mathjs": "4.4.2",
     "rlp": "2.2.4",
     "secp256k1": "3.8.0",
     "sha3": "2.1.1",
-    "solc": "0.4.18",
+    "solc": "^0.5.11",
```

Comment:
do this in development branch
without the ^

Reply:
Removed from commit tree.